configs/envoy_service_to_service.template.json

```diff
@@ -92,7 +92,7 @@
   "listeners": [
   {{ ingress_listener(9211) }},
```

Comment:
I did some research, but could not figure it out since I am not an expert. Maybe it's a bug of envoy, i leave a TODO here. Thanks.